lib/ansible/config/ansible_builtin_runtime.yml

```diff
@@ -1182,6 +1176,8 @@ plugin_routing:
       redirect: community.general.rax_scaling_group
     rax_scaling_policy:
       redirect: community.general.rax_scaling_policy
+    rpm_ostree_pkg:
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
Thanks! Added one. Please let me know if this is the right level of detail or if anything else needs to change.